uni/test/test_widget.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:flutter/material.dart';
+
+Widget testWidget(Widget child) {
+  return MaterialApp(
+      home: Scaffold(
+    body: child,
+  ));
+}
```

Comment:
What if this receives a list of providers, making it our mini test framework? Say:

```
Widget testableWidget(Widget child, List<ChangeNotifier> providers) {
  return MultiProvider(
    providers: providers
        .map((provider) => ChangeNotifierProvider.value(value: provider))
        .toList(),
    child: MaterialApp(
      home: Scaffold(
        body: child,
      ),
    ),
  );
}
```

Reply:
I like that idea and make the providers optional.
However, passing List<ChangeNotifier> isn't working and needed to make it List<ChangeNotifierProvider>

Reply:
What's the error? We should try to reduce the boilerplate as much as possible.

Reply:
Can you have a look at improving this current solution then? (What about cloning the providers inside the test method? Isn't the error related to provider disposing - GC?)